algorithms/src/fft/polynomial/dense.rs

```diff
@@ -256,6 +256,17 @@ impl<F: PrimeField> DensePolynomial<F> {
     }
 }
 
+impl<F: Field> From<super::SparsePolynomial<F>> for DensePolynomial<F> {
+    fn from(other: super::SparsePolynomial<F>) -> Self {
+        let mut result = vec![F::zero(); other.degree() + 1];
+        for (i, coeff) in other.coeffs {
+            result[i] = coeff;
+        }
```

Comment:
if `other.coeffs` is a vector, this could be simplified as `result[..other.coeffs.len()].copy_from_slice(&other.coeffs)`

Reply:
Unfortunately `other.coeffs` is a `Vec<(usize, F)>`, but `result` needs to be `Vec<F>`, so we can't directly copy the contents. In practice, the loops should execute at most 10 times, so it's not a perf concern.